clap_complete/tests/testsuite/common.rs

```diff
@@ -430,7 +430,7 @@ pub(crate) fn has_command(command: &str) -> bool {
         Ok(output) => output,
         Err(e) => {
             // CI is expected to support all of the commands
-            if is_ci() && cfg!(linux) {
+            if is_ci() && cfg!(target_os = "linux") {
```

Comment:
Huh, I thought I already ran check-cfg on this and fixed things.  I guess I overlooked clap.